src/pixi/IgePixiAnimation.js

```diff
@@ -89,10 +89,10 @@ var IgePixiAnimation = IgeClass.extend({
 	select: function (frames, fps = 15, loopCount, cellSheetAnimId, animName) {
 		var self = this;
 		if (
-			ige.entityTrack.trackEntityById[self._entity.entityId] &&
-			ige.entityTrack.trackEntityById[self._entity.entityId]._pixiContainer
+			ige.entitiesToRender.trackEntityById[self._entity.entityId] &&
```

Comment:
good eye :D

Reply:
instead of putting all related logic inside of a condition, let's invert the condition and return if false, will remove newly added indentation and make code more readable